op-e2e/system_test.go

```diff
@@ -273,6 +274,39 @@ func TestSystemE2EDencunAtGenesisWithBlobs(t *testing.T) {
 	require.Nil(t, err, "Waiting for safety of L2 block")
 }
 
+// TestCheckFjordScript ensures the op-chain-ops/cmd/check-fjord script runs successfully
+// against a test chain with the fjord hardfork enabled
+func TestCheckFjordScript(t *testing.T) {
```

Comment:
I suggest putting this into a new file like `checks_test.go`. We have a tendency in the op-e2e tests to put way too many unrelated tests in way too few files imo 😅